index.html

```diff
@@ -0,0 +1,250 @@
+     <a href="/someurl" class="sec3rlink32">JAVASCRIPT</a>
+ </li>
+</ul>
+<button type="submit" class="bb sec3rbutton sec3rlink4"><span class="d">See Projects</span></button>
+  </div>
+  <div class="right">        
+    <img src="images/d.png" alt="">
+  </div>
+</section>
+
+<!-- Sectoin 6 closed -->
+
+<!-- Section 7 started -->
+<section class="row6">
+  <div class="r3left">
+    <h2 class="tonic ">I’m Aziz Glad to see you!</h2>
+    <h4  class="paragraph1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h4>
+<h5 class="ets1" >ETS CONNECT</h5>
+<ul class="social-menue" >
+<li><img src="images/twitter.PNG" alt="abc"  class="lsls"></li>
+<li><img src="images/linkedinicon.PNG" alt="abc" class="lsls"></li>
+<li> <img src="images/medium.PNG" alt="abc" class="lsls"></li>
+<li> <img src="images/github.PNG" alt="abc" class="lsls"></li>
+<li><img src="images/angellist.PNG" alt="abc" class="lsls"></li>
+</ul>
+
+<button type="submit" class="bb sec3rlink6"><span class="d">See Projects</span></button>
+  </div>
+  
+
+  <div class="right">        
+    <h2 class="tonic sec3rlink31">Languages</h2>
+    <ul>
+      <li class="canopli6">
+         <a href="/someurl" class="sec3rlink316"><img src="images/jas.png" alt=""></a>
+         <a href="/someurl" class="sec3rlink316"><img src="images/css1.png" alt=""></a>
+          <a href="/someurl" class="sec3rlink316"><img src="images/n.png" alt=""></a>
+        
+       </li>
+    </ul>
+
+<ul class="ad">
+<li class="canopli2">
+  
+     <a href="/someurl" class="sec3rlink32">JAVASCRIPT</a>
+     <a href="/someurl" class="sec3rlink32">HTML</a>
+     <a href="/someurl" class="sec3rlink32 r6326">CSS</a>
+ </li>
+</ul>
+
+<ul>
+<li class="canopli6">
+ <a href="/someurl" class="sec3rlink316 ks"><img src="images/frame.PNG" alt=""></a>
+ <a href="/someurl" class="sec3rlink316"><img src="images/ks.png" alt=""></a>
+
+</li>
+</ul>
+
+  </div>
+</section>
+<!-- Section 7 closed -->
+
+<!-- Section 8 stated -->
+<section class="row7">
+    <div class="ro73">
+      <h2 class="head1">contact me</h2>
+      <p class="paragraph2">If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
+        I’d love to help with it</p>
+           <form action="https://formspree.io/f/xknegber" method="post" class="form2">
+              <input type="text" placeholder="Alexendra loremipsum" class="name1" minlength="3" maxlength="30" required>
+              <input type="email" placeholder="alexendra@gmail.com" class="em1" required>
+              <textarea name="text" id="ad" cols="30" rows="10" maxlength="500" class="message1" placeholder="Write your message here" required></textarea>
+               <button class="sub">Get It Touch</button>
+          </form>
+        </div>
+
+</section>
```

Comment:
Kindly make this section resemble Figma design as closely as possible for both mobile and desktop 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/187714919-72040e00-0b48-4530-ae94-2d77478161c9.png)
![image](https://user-images.githubusercontent.com/73714615/187714966-ff7a4bd8-bad5-4200-a638-ed886ce4f832.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/187715014-b94304c0-985c-4309-b518-3de6240d1fd7.png)
![image](https://user-images.githubusercontent.com/73714615/187715053-ef4494e0-a6d7-4f87-9d63-541e486baa06.png)

Reply:
Kindly, I recommend you to follow the suggestions of the previous reviewer, this will help your project to look more PRO 😉